tests/integration/test_integration.py

```diff
@@ -8,14 +8,17 @@
 """
 import tests.helpers as helpers
 
+from src.utils.config import config
+
 _TEST_EVENT = {
    "wsid": 55825,
    "ver": 15,
    "perm": None,
    "evtype": "NEW_VERSION",
    "objid": 2,
    "time": 101,
-   "user": "jayrbolton"
+   "user": "jayrbolton",
+   "index_runner_ver": ["1.9.10"],
```

Comment:
should this be updated to 1.9.11

Reply:
Well this is really just fake test data. I'm not sure it matters, but if we want it to always match the current version then we should load it from the config. I'll look at it a little